# DetectedObjects
# A list of all the detected object labels in an image.

Header header
Object[] objects
